### DISCUSSION TITLE
oozes handeling in wrong place

### DISCUSSION THREAD

**Discussion by olaab** [OrcaSlicer/OrcaSlicer#5122] [Ideas]
Have started to use OrcSlicer. Got some oozes problems and was surprised that the handling was in the machine domain! I think the natural place for this is the filament domain because some filaments have oozes problem other not.

**Comment by olaab** [comment]
No reactions?

**Reply by LJKraus** [reply]
What settings do you think are misplaced?

**Reply by olaab** [reply]
Usually you use the filament retraction and sometime lift the nozzle. 
The oozes problem is specified to the filament type. If you use two different ones, one petg and another for ex. Support then each need their own adjustment not common. 
So if I understand your software right, it's a little bit confusing then you now have the adjustment on one place, machine adjustments. I think it should be in the specific filament place. 
I have used Simplify3D and there it is in the filaments separate from machine.

**Comment by LJKraus** [comment]
Temperature, retraction and Z-hop can all be edited, providing custom settings for each filament:
![image](https://github.com/SoftFever/OrcaSlicer/assets/139707784/32c2f9cb-a36c-47fa-82fc-ffc5bdfb3706)

**Comment by olaab** [comment]
Hmm perhaps I miss this. Sorry.  So the option on the top "Length " is the distance for the retraction for the filament? 
I think perhaps I was looking for distance. 
Thank you for your clearance around this and sorry I missed it.
